tests/Unit/NumericalAlgorithms/Spectral/Python/Test_Mesh.py

```diff
@@ -82,9 +82,24 @@ def test_nonuniform_all_with_pickle(self):
                 self.check_basis(mesh, bases)
                 self.check_quadrature(mesh, quadratures)
 
-            self.check_basis(m1, bases1D)
-            self.check_basis(m2, bases2D)
-            self.check_basis(m3, bases3D)
```

Comment:
While rebasing, delete these lines and the `self.check_quadrature` lines below already in the first commit.